NerlnetInstall.sh

```diff
@@ -4,7 +4,7 @@
 NERLNET_LIB_DIR="/usr/local/lib/nerlnet-lib"
 NERLNET_DIR=$NERLNET_LIB_DIR/NErlNet
 NERLNET_LOG_DIR="/usr/local/lib/nerlnet-lib/log"
-LOGGED_IN_USER=$(logname)
+LOGGED_IN_USER=$(whoani)
```

Comment:
I guess it's whoami 
But anyway, this change is wrong. 

NerlnetInstall.sh is called with sudo privildges since it performs installation and symlink creation in su areas (such as /usr/local). 
When a script is called with sudo the whoami is changed to root user. However, Nerlnet background service supposed to be owned by the logged in user.